src/main/kotlin/no/nav/k9/domene/modell/OppgaveKø.kt

```diff
@@ -134,6 +135,10 @@ data class OppgaveKø(
             return true
         }
 
+        if (oppgaveKoder.isEmpty() && oppgave.aksjonspunkter.liste.isNotEmpty() || !oppgave.aksjonspunkter.hentAktive().keys.containsAll(oppgaveKoder)) {
```

Comment:
`tilhørerOppgaveTilKø` logikken litt kompleks, med alle mulig slags greier som kanskje, feks. kombinasjoner av kriterier, ekskludering og inkludering av enkelte. Det finnes noen enhetstester for som sjekker tilhørighet, så lag gjerne noen tester for denne logikken. Jeg tror de ligger i OppgaveKøTest, kan f.eks. se på tester knyttet til merknad.

Reply:
Bra tips! Lager tester for det